.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestBase.cs

```diff
@@ -320,6 +320,11 @@ public void TestDateTimeMergedParser()
             }
         }
 
+        public void TestSequence()
+        {
+            TestNumber();
```

Comment:
If TestNumber can be used in NumberModel,OrdinalModel,PercentModel,NumberRangeModel and PhoneNumberModel, I suggest to rename TestNumber to more general method(such as TestBasic?). And not need to create a new method TestSequence

Reply:
Is NumberWithUnit also using the same TestNumber method? If so, I agree with @yiylin.

Reply:
No, but the NumberWithUnit's test code is very similar to TestNumber, and I think after adding a "if" to TestNumber , then NumberWithUnit also can use this method to test

Reply:
Let's keep this one like this then. In the future there will be more cases where there will be differences from sequences to numbers.
But I'd rename TestSequence to TestPhone as this is specific to it.